webrtc.html

```diff
@@ -6817,9 +6820,10 @@ <h2>Dictionary RTCRtpContributingSource Members</h2>
             "idlMemberType"><a>DOMHighResTimeStamp</a></span>, required</dt>
             <dd>
               <p>The timestamp of type DOMHighResTimeStamp [[!HIGHRES-TIME]],
-              indicating the most recent time of playout of an RTP packet
-              containing the source. The timestamp is defined in
-              [[!HIGHRES-TIME]] and corresponds to a local clock.</p>
+              indicating the most recent time of playout of media
+              originating from this source. The timestamp is defined as
```

Comment:
This rephrasing lost the reference to the RTP packet. Was this intentional?
I'm not sure it contributes any value (is there media not contained in an RTP packet?), but we might consider "the most recent time of playout of media that arrived in an RTP packet originating from this source".
This may be "interesting" to compute in the case of media that's mixed before playout, but let's burn that bridge in another bug.